libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/changeSharingHandler.ts

```diff
@@ -81,6 +97,13 @@ function* saveSharing(
         saveSharingCtx,
     );
 
+    if (saveSharingCtx.cmd.payload.newShareProps.granteesToAdd.length !== 0) {
+        yield call(addGrantees, ctx, saveSharingCtx);
+    }
+    if (saveSharingCtx.cmd.payload.newShareProps.granteesToDelete.length !== 0) {
+        yield call(removeGrantees, ctx, saveSharingCtx);
+    }
```

Comment:
small stuff, please destructure the ctx/payload etc so that these conditions don't so many dosts.

also, please change to !isEmpty()

Reply:
Done